packages/mds-agency-cache/index.ts

```diff
@@ -181,16 +180,15 @@ async function wipeDevice(device_id: UUID) {
     decorateKey(`device:${device_id}:device`)
   ]
   if (keys.length > 0) {
-    logger.info('mds-agency-cache::wipeDevice, deleting keys', { keys })
+    AgencyCacheLogger.debug('mds-agency-cache::wipeDevice, deleting keys', { keys })
```

Comment:
Ditto. Should be able to remove all the prefixes from the log text.

Reply:
Fixed in a7334b7